Unable to use a profile with assume role

Hi,

I'm getting the following error when trying to use a profile with a source_profile to assume role:
Caused by: com.amazonaws.SdkClientException: To use assume role profiles the aws-java-sdk-sts module must be on the class path

The full stack trace can be found here:
https://gist.github.com/JoshiiSinfield/ddb945ddedfe1eb9c4d3c67b9de6fde0

Cheers,
Josh

--- a/pom.xml
+++ b/pom.xml
@@ -72,6 +72,12 @@
             <version>${awssdk.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-sts</artifactId>
+            <version>${awssdk.version}</version>
+        </dependency>
+
         <!-- Spring -->
         <dependency>
             <groupId>org.springframework.boot</groupId>

